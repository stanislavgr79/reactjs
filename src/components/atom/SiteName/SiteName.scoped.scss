p {
    color: #F65261;
    margin: auto;
}

.site_name {
    font-size: 22px;
}