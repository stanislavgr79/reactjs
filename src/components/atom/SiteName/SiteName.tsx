import React from 'react';
import './SiteName.scoped.scss';

const Sitename = (): JSX.Element => {
  return (
    <p className="site_name">
      <strong>netflix</strong>roulette
    </p>
  );
};

export default Sitename;
