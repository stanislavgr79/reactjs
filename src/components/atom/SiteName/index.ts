import SiteName from './SiteName';

export default SiteName;
