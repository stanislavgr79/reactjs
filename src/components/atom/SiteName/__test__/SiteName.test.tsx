import React from 'react';
import { render } from '@testing-library/react';

import SiteName from '../SiteName';

describe('SiteName', () => {
  test('snapshot renders', () => {
    const tree = render(<SiteName />);

    expect(tree.baseElement).toMatchSnapshot();
    expect(tree.baseElement).toContainHTML('<p class="site_name">');
  });
});
