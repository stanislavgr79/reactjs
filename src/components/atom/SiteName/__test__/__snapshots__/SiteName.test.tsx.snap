// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SiteName snapshot renders 1`] = `
<body>
  <div>
    <p
      class="site_name"
    >
      <strong>
        netflix
      </strong>
      roulette
    </p>
  </div>
</body>
`;
